Cargo.toml

```diff
@@ -10,11 +10,11 @@ repository = "https://github.com/MagicalBitcoin/rust-hwi"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bitcoin = { version = "0.25", features = ["use-serde", "base64"] }
-serde = { version = "^1.0", features = ["derive"] }
-serde_json = { version = "^1.0" }
-strum_macros = "0.18.0"
-base64 = "^0.11"
+bitcoin = { version = "0.28.1", features = ["use-serde", "base64"] }
```

Comment:
I think it this works similarly. https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
It would work similarly. See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio. Though I changed it as you said.

Reply:
Nope, I was wrong, we get `0.28.2` even if we specify bitcoin like this. Solving :)

Reply:
Ok great, thanks! :D